srcpkgs/i3/template

```diff
@@ -28,3 +28,9 @@ i3-devel_package() {
 		vmove usr/include
 	}
 }
+
+i3-gaps_package() {
```

Comment:
will this allow i3-gaps updates to this new release? or will just deprecate i3-gaps and install i3?